docker-compose.ci.yml

```diff
@@ -1,13 +1,16 @@
 version: "3.8"
 services:
+  activator-aviris-l2:
+    image: "activator-aviris-l2:${GIT_COMMIT:-latest}"
+
   terraform:
-    image: quay.io/azavea/terraform:0.13.5
+    image: quay.io/azavea/terraform:0.14.3
     volumes:
       - ./:/usr/local/src
       - $HOME/.aws:/root/.aws:ro
     environment:
-      - AWS_PROFILE=${AWS_PROFILE:-nasa-hsi}
-      - GIT_COMMIT=${GIT_COMMIT:-latest}
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
```

Comment:
I had to remove the `AWS_PROFILE` variable because it was causing the access key pair to be ignored. I'm open to any ideas on how we could accommodate both, as the `AWS_PROFILE` passthrough and default is useful for local work.

Reply:
We discussed some of the pros and cons of different approaches and the desire to minimize the surface area of any change we make as much as possible.

I decided that the smallest change would be to unset the environment variable in the GitHub Actions workflow itself and leave a comment explaining what was happening. I'm also going to look into opening in issue on the AWS CLI about this.